client/src/components/InputDescription.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { InputTextarea } from 'primereact/inputtextarea';
+
+export function InputDescription(props) {
+  return (
+    <InputTextarea
+      value={props.value}
+      onChange={props.onValueChange('description')}
```

Comment:
I don't think I understand this. Is this a closure? Just keep it as plain function